changelog/changelog.go

```diff
@@ -5,28 +5,28 @@ import (
 )
 
 const (
-	tplStr = `{{.OldRelease}} -> {{.NewRelease}}
+	tplStr = `### {{.OldRelease}} -> {{.NewRelease}}
```

Comment:
The levels of heading were selected to remain uniform with our existing changelogs.